tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -21,7 +21,13 @@ use testapi;
 use Utils::Architectures;
 
 sub get_script_run {
-    my $pre_test_cmd         = "/usr/share/qa/tools/test_virtualization-guest-upgrade-run";
+    my $pre_test_cmd = "";
+    if (is_s390x) {
+        $pre_test_cmd = "/usr/share/qa/virtautolib/lib/test_full_guest_upgrade.sh";
+    }
+    else {
+        $pre_test_cmd = "/usr/share/qa/tools/test_virtualization-guest-upgrade-run";
+    }
```

Comment:
Could you help add explanatory comments here about why s390x use different script ?

Reply:
yep, good point.  had added some explanatory comments to explain why use with the different script - `test_full_guest_upgrade.sh `for s390x arch here.